### ISSUE TITLE
Update map.json?

### ISSUE THREAD

**Issue by artxtra** [aaronpk/emoji-detector-php#8]
Hi,
It's not really an issue but I wonder if you frequently update map.json with new unicode emojis?
I use your lib in my science project and I need to parse every emojis I can find in Twitter.
BTW, thanks, this script is trully awesome for me :)

**Comment by aaronpk** [comment]
Thanks! I do update it from time to time. I'm overdue for an update right now though!

**Comment by gregavola** [comment]
@aaronpk Any update on this?

**Comment by aksafan** [comment]
Hello @aaronpk

Any update incoming? Would be nice to use your very helpful library with the newest emojis available  =) 

Thanks!

**Comment by isupremedyou** [comment]
@aaronpk Would you be able to update `map.json` with the most recent set of emojis?

**Comment by Mahdi-Elvis** [comment]
Hi. Thank you for your great library.
is There any update coming ?

**Comment by janisblaus** [comment]
Props for update as well <3

**Comment by daneren2005** [comment]
I echo that it would be wonderful to have an updated list of emoji!

**Comment by daneren2005** [comment]
In case anyone else cares, this was super easy to just add as a build step since they included the build script in the repo:

`exec('cd ' . getRootDir()  . '/vendor/p3k/emoji-detector && php build/generate-map.php')`

**Comment by aksafan** [comment]
> In case anyone else cares, this was super easy to just add as a build step since they included the build script in the repo and it is extremely fast:
> 
> `exec('cd ' . getRootDir() . '/vendor/p3k/emoji-detector && php build/generate-map.php')`
> 
> Realistically I think I like this better since that is pulling from something which is consistently updated.

Actually don't like the idea of using `exec` - a bit paranoid about this function))

Frankly speaking I've just forked this library and made some upgrades (if somebody intresting in it is [here](https://github.com/aksafan/emoji-detector-php/releases)).

**Comment by daneren2005** [comment]
Fair point.  I'm just moving the generation script into my own local build step.  I hadn't thought that through but if you just blindly call that function that leaves you open to a fairly easy supply channel attack.

**Comment by aaronpk** [comment]
I will happily accept a PR with updates or with something to automate this!
